packages/components/psammead-caption/src/index.jsx

```diff
@@ -36,6 +36,9 @@ const Caption = styled.figcaption`
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
     padding: ${GEL_SPACING} 0;
```

Comment:
As figure [has 24 padding already](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-figure/src/index.jsx#L6), I think caption needs to have zero bottom padding at all breakpoints